### PR TITLE
28 split view test fixture

### PR DESCRIPTION
**Pull request by blackbaud-svossen**
This PR addresses issue #28 

Added a split view test fixture which leverages the existing country field test fixture

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-28-split-view-test-fixture](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-28-split-view-test-fixture)

File: src/app/public/testing/split-view/fixtures/split-view-fixture-test.component.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<div
+  class="sky-padding-even-large"
+>
+  <sky-split-view
+    data-sky-id="test-split-view"
+    [messageStream]="splitViewStream"
+    [backButtonText]="backButtonText"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/testing/split-view/fixtures/split-view-fixture-test.component.html:6]: Alphabetical ordering.

File: src/app/public/testing/split-view/split-view-fixture-workspace.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export interface SkySplitViewFixtureWorkspace {
+  ariaLabel: string;
+  backButtonText: string;
+  backButtonIsVisible: boolean;
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/testing/split-view/split-view-fixture-workspace.ts:3]: Alphabetical ordering.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Just a few small things.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-28-split-view-test-fixture](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-28-split-view-test-fixture)

**Pull request merged**